openai/azure-openai/runtime/src/main/java/io/quarkiverse/langchain4j/azure/openai/AzureOpenAiChatModel.java

```diff
@@ -116,7 +120,8 @@ private Response<AiMessage> generate(List<ChatMessage> messages,
                 .topP(topP)
                 .maxTokens(maxTokens)
                 .presencePenalty(presencePenalty)
-                .frequencyPenalty(frequencyPenalty);
+                .frequencyPenalty(frequencyPenalty)
+                .responseFormat(responseFormat);
```

Comment:
Looking at the Lang4Chain documentation, I see

```java
AzureOpenAiChatModel.builder()
        ...
        .responseFormat(new ChatCompletionsJsonResponseFormat())
        .build();
```

I don't see, how the correct item is inject in the builder here, is it actually working as expected ?

Reply:
Well, yes, it works as Expected. ChatCompletionsJsonResponseFormat is also possible, but you lose the flexibility of changing it over the application properties. It is now implemented the same way as for the open ai part directly.

Reply:
Thanks !